Godeps/Godeps.json

```diff
@@ -7,9 +7,8 @@
 	],
 	"Deps": [
 		{
-			"ImportPath": "github.com/Sirupsen/logrus",
-			"Comment": "v1.0.0",
-			"Rev": "202f25545ea4cf9b191ff7f846df5d87c9382c2b"
+			"ImportPath": "github.com/sirupsen/logrus",
```

Comment:
We lost version info here. maybe you should `git checkout 1.0.2` in logrus gopath directory and then update it in runtime-tools.